src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -169,7 +171,17 @@ public static String composeRpcMethodHeaderSampleCode(
       Method method,
       List<MethodArgument> arguments,
       TypeNode clientType,
-      Map<String, ResourceName> resourceNames) {
+      Map<String, ResourceName> resourceNames,
+      Map<String, Message> messageTypes) {
+    if (method.isPaged()) {
+      Field pagedItemField =
+          messageTypes.get(method.outputType().reference().name()).fields().get(0);
+      TypeNode pagedItemType =
+          TypeNode.withReference(pagedItemField.type().reference().generics().get(0));
+      return SampleCodeWriter.write(
+          MethodSampleCodeHelperComposer.composeUnaryPagedRpcMethodSampleCode(
```

Comment:
As in #573, please inline the logic in this class instead of a separate one.

Reply:
SG